docs/jenkins_gerrit_integration.md

```diff
@@ -0,0 +1,203 @@
+
+## <a name="gerrit-plugin-setup"></a> Gerrit Trigger plugin setup
+If Gerrit Trigger plugin is installed correctly, its settings
+should be available under Manage Jenkins.
+At Gerrit Connection Setting a working connection must be defined
+to the Gerrit server (can be tested with Test Connection). Note that 
+the SSH private key's public pair should have been already added to Gerrit.
+
+In the Advanced settings, at Gerrit Reporting Values, the default
+reporting values can be configured. These can be overwritten later in the job's
+configuration. In this guide the Gerrit review message is crafted by hand,
+therefore it is strongly advised to remove all the default values from the list
+(otherwise once the review message is sent, the job will send another
+reporting value based on this default configuration that can overwrite 
+the first review and change the final result shown in Gerrit).
+
+In the REST API section the user performing the Gerrit review must be added,
+with Enable Code-Review and Enable Verified set to enabled.
+
+Here is the config that worked best for us:
+![Reporting values](/docs/images/gerrit_jenkins/gerrit_plugin_values.png)
+![REST API](/docs/images/gerrit_jenkins/gerrit_plugin_restapi.png)
+
+## <a name="configuring-the-jenkins-job"></a> Configuring the Jenkins job
+The job is configured as a freestyle project.
+### <a name="jenkins-source-code-management"></a> Source Code Management
+In the job's configuration menu set Source Code Management to Git.  The
+Repository URL is the link of the repo that has to be checked. Add new or set
+from the list the credentials of the user who can clone and review the changes.
+In the Repositories' advanced settings, set refspec to `${GERRIT_REFSPEC}`.
+Branch Specifier should be set to `**`.
+
+In the Additional Behaviour section, add Strategy for choosing what to build,
+then select the Gerrit Trigger from the Choosing strategy list.
+
+If needed, the subdirectory used to check out the repo could be also defined
+as an Additional Behaviour.
+
+![Source Code Management configuration](/docs/images/gerrit_jenkins/source_code_management.png)
+
+### <a name="jenkins-gerrit-trigger"></a> Gerrit Trigger
+Build Triggers should be set to Gerrit event. This opens the Gerrit Trigger
+configuration form.
+In the Choose a Server field, the server that has been already configured in 
+[Gerrit Trigger plugin setup](#gerrit-plugin-setup) should be selected from the list.
+In the server's advanced section, the Gerrit reporting values
+(e.g. Succesful - verified +1) and Build Messages can be configured.
+If left empty, values configured in the [plugin's setup](#gerrit-plugin-setup)
+are used. One possible setting is to leave all fields empty, 
+except Verify - Failed - -1. This ensures that the values reported at the 
+end of the analysis (crafted manually) will not be overwritten by Jenkins.
+
+Add a Trigger on Patchset Created and tick Exclude No Code Change (there is no
```

Comment:
Please highlight plugin configuration value names in text like `Exclude No Code Change`